src/main/java/plugin/TuebIlias.java

```diff
@@ -40,17 +41,17 @@ public LoginStatus login(String username, String password) {
 
 			Document doc = Jsoup.parse(html);
 			Element form = doc.select("form[action*=idp").first();
-			Element input_tag = form.select("input[name='lt']").first();
-			String ltKey = input_tag.attr("value");
-			
+
+			// Updated 06.02.2020 to fit the new login process
 			post = new HttpPost("https://idp.uni-tuebingen.de" + form.attr("action"));
-			nvps.add(new BasicNameValuePair("username", username));
-			nvps.add(new BasicNameValuePair("password", password));
-			
-			nvps.add(new BasicNameValuePair("lt", ltKey));
+			nvps.add(new BasicNameValuePair("j_username", username));
+			nvps.add(new BasicNameValuePair("j_password", password));
+
 			nvps.add(new BasicNameValuePair("execution", "e1s1"));
-			nvps.add(new BasicNameValuePair("_eventId", "submit"));
+			nvps.add(new BasicNameValuePair("donotcache", "1"));
+			nvps.add(new BasicNameValuePair("_eventId_proceed", ""));
 			post.setEntity(new UrlEncodedFormEntity(nvps, Consts.UTF_8));
```

Comment:
i guess i will blindly trust you with that changes :) thx!

Reply:
Hehe, it works, I promise. At least it does for me now ;) 
Not sure with the `nvps.add(new BasicNameValuePair("donotcache", "1"));` though. It tells the server to not set a cookie I think. Did not test it. Set it to 1 and it worked^^